debian/postinst

```diff
@@ -59,13 +59,12 @@ case "$1" in
             JIGASI_SIPPWD=$(echo -n $RET | base64 -)
 
             # 8-chars random secret, alternative to pwgen 8
-            JIGASI_SECRET=`head -c 8 /dev/urandom | tr '\0-\377' 'a-zA-Z0-9a-zA-Z0-9a-zA-Z0-9a-zA-Z0-9@@@@####'`
+            JIGASI_XMPP_SECRET=`head -c 8 /dev/urandom | tr '\0-\377' 'a-zA-Z0-9a-zA-Z0-9a-zA-Z0-9a-zA-Z0-9@@@@####'`
```

Comment:
In other projects we use:
```
function generateRandomPassword() {
    cat /dev/urandom | tr -dc 'a-zA-Z0-9' | head -c 16
}
```

I can do that in a follow up PR if you prefer not to do it here.

Reply:
👍